### PR TITLE
Feature/payoutstepper

### PR DESCRIPTION
**Pull request by dhruv-divtech**

### CODE REVIEW

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    padding: '0px'
+  },
+  '& .MuiStepIcon-text': {
+    display: 'none'
+  },
+  '& .MuiStepLabel-alternativeLabel': {
+    color: theme.palette.text.secondary
+  },
+  '& .MuiStepLabel-label.MuiStepLabel-alternativeLabel': {
+    marginTop: '8px'
```
Comment (WorkspaceChetan) [inline src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx:23]: use theme.spacing

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  },
+
+  '& .MuiStepContent-root': {
+    borderLeft: '0',
+    fontWeight: 500,
+    fontSize: '12px',
+    fontStyle: 'normal',
+    lineHeight: '140%',
+    letterSpacing: '0.3px',
+    paddingLeft: '13px',
```
Comment (WorkspaceChetan) [inline src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx:33]: use theme.spacing

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  },
+  '& .mui-15shpc-MuiStepLabel-root .MuiSvgIcon-root': {
+    width: '16px',
+    height: '16px'
+  },
+  '& .mui-style-1fu8gtm-MuiSvgIcon-root-MuiStepIcon-root.Mui-completed': {
+    color: theme.palette.text.secondary
+  },
+  '& .mui-1pe7n21-MuiStepConnector-root': {
+    marginLeft: '4px',
```
Comment (WorkspaceChetan) [inline src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx:50]: use theme.spacing

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    marginTop: '-17px'
```
Comment (WorkspaceChetan) [inline src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx:51]: use theme.spacing

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx (hunk 2 above)
Comment (dhruv-divtech) [reply src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx]: resolved

File: src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx (hunk 1 above)
Comment (dhruv-divtech) [reply src/components/UIComponents/UIVerticalStepper/UIStyledVerticalStepper.styled.tsx]: resolved

**Pull request merged**